packages/hash/frontend/src/pages/[account-slug]/types/entity-type/property-list-card.tsx

```diff
@@ -182,57 +186,204 @@ export const PropertyTypeRow = ({
   propertyIndex: number;
   onRemove: () => void;
 }) => {
-  const { watch } = useFormContext<EntityTypeEditorForm>();
+  const { control, watch } = useFormContext<EntityTypeEditorForm>();
+  const { update } = useFieldArray({
+    control,
+    name: "properties",
+  });
+
   const propertyTypes = usePropertyTypes();
-  const propertyId = mustBeVersionedUri(
-    watch(`properties.${propertyIndex}.$id`),
-  );
+  const [anchorEl, setAnchorEl] = useState<HTMLDivElement | null>(null);
+  const [multipleValuesMenuOpen, setMultipleValuesMenuOpen] = useState(false);
+
+  const draftProperty = watch(`properties.${propertyIndex}`);
+
+  const { $id, array, minValue, maxValue, required } = draftProperty;
+
+  const propertyId: VersionedUri = mustBeVersionedUri($id);
   const property = propertyTypes ? propertyTypes[propertyId] : null;
 
-  if (!property) {
-    throw new Error("Missing property type");
-  }
+  // if (!property) {
+  //   throw new Error("Missing property type");
+  // }
 
-  return (
-    <TableRow>
-      <TableCell>
-        <Typography variant="smallTextLabels" fontWeight={500}>
-          {property.title}
-        </Typography>
-      </TableCell>
-      <TableCell>
-        <PropertyExpectedValues property={property} />
-      </TableCell>
-      <TableCell sx={{ textAlign: "center" }}>
-        <Checkbox />
-      </TableCell>
-      <TableCell sx={{ textAlign: "center" }}>
-        <Checkbox />
-      </TableCell>
-      <TableCell sx={{ px: "0px !important" }}>
-        <TextField
-          placeholder="Add default value"
-          sx={{
-            width: "100%",
-            [`.${tableRowClasses.root}:not(:hover) & .${outlinedInputClasses.root}:not(:focus-within)`]:
-              {
-                boxShadow: "none",
-                [`.${outlinedInputClasses.notchedOutline}`]: {
-                  borderColor: "transparent",
-                },
-                [`.${outlinedInputClasses.input}::placeholder`]: {
-                  color: "transparent",
+  return property ? (
+    <>
+      <TableRow>
+        <TableCell>
+          <Typography variant="smallTextLabels" fontWeight={500}>
+            {property.title}
+          </Typography>
+        </TableCell>
+        <TableCell>
+          <PropertyExpectedValues property={property} />
+        </TableCell>
+        <TableCell
```

Comment:
Can we make `MultipleValuesCell` component and put this stuff in there? Will make this component cleaner and could also help move the `watch` call lower down the component tree (if its at all necessary). It doesn't matter where in the tree you put the popper, so you can put the popper in `MultipleValuesCell` too.

Reply:
Great suggestion! Done in 669f2a5700d2e0a6385302b661eb077a24186144.